<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkEndoscopyCalibrationViewControls</class>
 <widget class="QWidget" name="QmitkEndoscopyCalibrationViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>716</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkEndoscopyCalibration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="m_HelpText">
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Steps to calibrate your camera with your tracking system:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;1. Start&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;2. Load Camera Intrinsics&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;3. Take a Screenshot&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;4. Login all Chessboard-Points&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;You can select a point by clicking it on the Chessboard-Scheme. The order of chessboard-points is based on the perspective of your camera.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;5. Calibrate&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;6. Show points in Video and change position of your camera&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QmitkChessboardSchemeWidget" name="m_MeasurementSchemeChessboard" native="true">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="m_TrackingToolOutputLayout">
     <item>
      <widget class="QmitkToolTrackingStatusWidget" name="m_ToolStatusWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Start Tracking, Capturing and Calibration:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_StartButton">
         <property name="toolTip">
          <string>start</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_StopButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>stop</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="m_LoadIntrinsics">
     <property name="text">
      <string>Load Camera Intrinsics</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Calibration:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_TakeScreenshotButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Take Screenshot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_LoginPointOnChessboardButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Login Point</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_Calibrate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Calibrate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Visualization:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_ShowPointInVideo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show Point in Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_ShowFigureInVideo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show Figure in Video</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkChessboardSchemeWidget</class>
   <extends>QWidget</extends>
   <header>src/internal/QmitkChessboardSchemeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkToolTrackingStatusWidget</class>
   <extends>QWidget</extends>
   <header>qmitktooltrackingstatuswidget.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">QmitkDataStorageComboBox.h</include>
  <include location="local">QmitkToolTrackingStatusWidget.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
